OIPS/oip-draft-versioning.md

```diff
@@ -0,0 +1,41 @@
+---
+oip: 4
+title: Releases Versioning Specification
+author:  Francesco Pacella <f@ost.com>
+discussions-to: -
+status: Draft
+type: Meta
+category -
```

Comment:
Suggest striking category, requires, and replaces from the header, as these are optional and empty.